test/Polytopes/runtests.jl

```diff
@@ -1,6 +1,11 @@
 const pm = Polymake
 
 using Oscar
+
+if (haskey(ENV, "GITHUB_ACTIONS"))
+    Polymake.shell_execute(raw"$Verbose::cpp=3;")
```

Comment:
What is this about?

Reply:
This is a debug flag from polymake to add some debugging output when polymake wrappers are compiled during the tests. It was helpful to identify one more GLIBCXX issue. I think it should stay in there to make it easier to analyze other issues with polymake compilation at runtime (we also have this in the Polymake.jl tests).

Reply:
That's fine by me, but then a comment should be added to the code code explaining this